router/smoke/openshift_2/conftest.py

```diff
@@ -107,4 +95,21 @@ def sender(request, iqa) -> Union[SenderJava, SenderPython, SenderNodeJS]:
         s = request.param.split('_')
         sender_implementation = s[1]
         sender_number = int(s[2])
-        return iqa.get_clients(Sender, sender_implementation)[sender_number]
+        sender = iqa.get_clients(Sender, sender_implementation)[0]
+        sender.set_url("amqp://%s:5672/address" % request.config.option.cluster[sender_number])
+        return sender
+
+
+@pytest.fixture()
+def router_cluster(request, iqa) -> Tuple[Dispatch, str, str]:
+    """
+    Returns the router, cluster and token from parameter
+    :param request:
+    :param iqa:
+    :return: Returns router instance based on parametrized info
+    """
+    if 'router_' in request.param:
+        router_number = int(request.param.split('_')[1])
+        return (iqa.get_routers()[router_number],
+                request.config.option.cluster[router_number],
+                request.config.option.token[router_number])
```

Comment:
Here I am returning a tuple now (which includes router instance, cluster ip and token)